Pickers/AbstractActionSheetPicker.m

```diff
@@ -101,20 +100,15 @@ - (instancetype)init
     {
         self.presentFromRect = CGRectZero;
         self.popoverBackgroundViewClass = nil;
-
         self.supportedInterfaceOrientations = (UIInterfaceOrientationMask) [[UIApplication sharedApplication]
-                                                                                           supportedInterfaceOrientationsForWindow:
-                                                                                                   [UIApplication sharedApplication].keyWindow];
-
+                                                                            supportedInterfaceOrientationsForWindow:
+                                                                            [UIApplication sharedApplication].keyWindow];
         UIBarButtonItem *sysDoneButton = [self createButtonWithType:UIBarButtonSystemItemDone target:self
                                                              action:@selector(actionPickerDone:)];
-
         UIBarButtonItem *sysCancelButton = [self createButtonWithType:UIBarButtonSystemItemCancel target:self
                                                                action:@selector(actionPickerCancel:)];
-
         [self setCancelBarButtonItem:sysCancelButton];
         [self setDoneBarButtonItem:sysDoneButton];
-
```

Comment:
Thanks, almost done.
But -now you actually delete whole string, not only whitespace on it (e.g .\n symbol). Please, try to put it back. You can look here differences: https://github.com/skywinder/ActionSheetPicker-3.0/pull/140/files And here is a red lines. Please, remove only whitespaces, not empty strings. I think the problem in some auto-format settings in your IDE.